request_test.go

```diff
@@ -42,6 +42,36 @@ func TestGet(t *testing.T) {
 	})
 }
 
+
+func TestHeaders(t *testing.T) {
+	t.Run("should add headers to request", func(t *testing.T) {
+		cli := requestfy.NewClient()
+
+		headers := map[string]string{
+			"bar":  "foo",
+			"cool": "header",
+		}
+
+		r := cli.Request()
+
+		for k, v := range headers {
+			r.SetHeader(k, v)
+		}
+
+		for k, v := range r.GetHeaders() {
+			aimVal, ok := headers[k]
```

Comment:
What does "aimVal" mean?

Reply:
change v = wantedVal and aimVal changed for currentVal